cfgov/ask_cfpb/models/pages.py

```diff
@@ -586,22 +554,81 @@ class AnswerPage(CFGOVPage):
         on_delete=models.SET_NULL,
         related_name='redirected_pages',
         help_text="Choose another Answer to redirect this page to")
+    redirect_to_page = models.ForeignKey(
+        'self',
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name='redirect_to_pages',
+        help_text="Choose another AnswerPage to redirect this page to")
+    featured = models.BooleanField(
+        default=False,
+        help_text=(
+            "Check to make this one of two featured answers "
+            "on the landing page."))
+    featured_rank = models.IntegerField(blank=True, null=True)
+    category = models.ManyToManyField(
```

Comment:
I'll remove this -- this is just from before when we thought we'd migrate everything over first.  Once we get the final mapping from CE we can open up another PR to handle categories and subcategories.